tabs/osd.js

```diff
@@ -463,6 +473,11 @@ OSD.chooseFields = function () {
           ]);
         }
       }
+      if (semver.gte(CONFIG.flightControllerVersion, "3.2.0")) {
+        OSD.constants.DISPLAY_FIELDS = OSD.constants.DISPLAY_FIELDS.concat([
```

Comment:
OK, I didn't know and `CONFIG.flightControllerVersion` is all over the place in the code. What is the reason this can not be used anymore?
Checking for the `CONFIG.apiVersion` is also strange for this feature as it didn't require the api to change. It could be the case that the same api version on a different flightcontrollerversion doesn't have a feature because the api didn't change.
But since we did not release 3.2.0 yet, this isn't a problem for now. I will change this to `CONFIG.apiVersion` equal or higher than `3.16`